kura/examples/org.eclipse.kura.example.gpio/OSGI-INF/gpio.xml

```diff
@@ -26,6 +26,7 @@
    <property name="service.pid" type="String" value="org.eclipse.kura.example.gpio.GpioComponent"/>
    <service>
        <provide interface="org.eclipse.kura.example.gpio.GpioComponent"/>
```

Comment:
@ctron what about removing the nonsense export ``org.eclipse.kura.example.gpio.GpioComponent``?
I doesn't make sense to me.
Same for the other examples.